test/built-ins/TypedArrays/ctors-bigint/typedarray-arg/other-ctor-returns-new-typedarray.js

```diff
@@ -14,8 +14,8 @@ includes: [testBigIntTypedArray.js]
 features: [BigInt, TypedArray]
 ---*/
 
-var sample1 = new BigInt64Array();
-var sample2 = new BigUint64Array();
+var sample1 = new BigInt64Array(7);
+var sample2 = new BigUint64Array(7);
```

Comment:
while this is great, the description message is useless here. Can you fix this one too?